src/drive/lib/doctypes.js

```diff
@@ -15,6 +16,10 @@ export const schema = {
       old_versions: {
         type: 'has-many',
         doctype: 'io.cozy.files.versions'
+      },
+      encryption: {
+        type: 'io.cozy.files:has-many',
```

Comment:
do we really need `io.cozy.files:` here? It's not just `has-many`?

Reply:
No, `io.cozy.files:has-many` is the expected name for HasManyFiles: https://github.com/cozy/cozy-client/blob/master/docs/relationships.md#files-relations